packages/react-components/babel-preset-storybook-full-source/src/index.d.ts

```diff
@@ -0,0 +1,11 @@
+export interface DependencyEntry {
+  /**
+   * Replaces the dependency with another
+   * @default \@fluentui/react-components
+   */
+  replace: string;
+}
+
+export type BabelPluginOptions = Record<string, DependencyEntry>;
+
+export function fullSourcePlugin(babel: typeof Babel, options: BabelPluginOptions): Babel.PluginObj;
```

Comment:
Trick we are using in all scripts/ packages - completely mitigates duplication

```suggestion
export { fullSourcePlugin } from './index';
export type { BabelPluginOptions } from './index';
```